test/src/pages/details/book-details-context.js

```diff
@@ -15,8 +23,10 @@ function BookDetailsWithContext({data, children}) {
 }
 
 export default function BookDetailsLoader({slug, children}) {
+    const absoluteSlug = slug.startsWith('/') ? slug : `/${slug}`;
```

Comment:
I can't tell, does this branch get tested by the dual-view test? I don't see an expect for it

Reply:
That's just a test helper function, but it doesn't look like any testing code actually does come in with a slug beginning with a slash, so it's an unneeded conditional.